package.json

```diff
@@ -170,7 +173,15 @@
     "lodash": "4.17.21",
     "axios": "0.21.1",
     "marked": "0.7.0",
-    "node-fetch": "2.6.1"
+    "node-fetch": "2.6.1",
+    "graphql": "15.8.0",
+    "@graphql-typed-document-node/core": "3.2.0"
+  },
+  "comments": {
+    "on_resolutions": {
+      "graphql": "Having graphql fixed at 15.8.0 allows apollo codegen and graphql-codegen to work together. Once the former is deprecated, you can remove this",
+      "typed_document_node": "https://github.com/dotansimha/graphql-code-generator/issues/9293"
+    }
```

Comment:
Trying something here - I wanted to document a few things that I did in the package.json, but since it's just JSON, it doesn't support actual comments, and some places online recommended just throwing a custom object in there for stuff like this!